tests/acceptance/01_vars/02_functions/getgroups3.cf

```diff
@@ -0,0 +1,81 @@
+#######################################################
+#
+# Test getgroups() arg0 and arg1
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };   
+      version => "1.0";
+}
+
+bundle agent init
+{
+  vars:
+      # All groups except root, bin, and daemon
```

Comment:
I think the comment here is wrong. This would be root, bin, and daemon so long as non of them have uid 0, 1 or 2.